# DP: Description: Make glibc-2.3.5 compile with gcc-3.4/4.0 + binutils 2.16
#		   on m68k fixed by adding #APP.
# DP: Related bugs: 
# DP: Dpatch author: GOTO Masanori <gotom@debian.org>
# DP: Patch author: Andreas Schwab <schwab@suse.de>
# DP: Upstream status: Pending
# DP: Status Details: 
# DP: Date: 2005-08-03

From: Andreas Schwab <schwab@suse.de>

---
 include/libc-symbols.h |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/include/libc-symbols.h
+++ b/include/libc-symbols.h
@@ -162,12 +162,12 @@
 #define __make_section_unallocated(section_string)	\
   asm (".section " section_string "\n\t.previous");
 
-/* Tacking on "\n\t#" to the section name makes gcc put it's bogus
+/* Tacking on "\n#APP\n\t#" to the section name makes gcc put it's bogus
    section attributes on what looks like a comment to the assembler.  */
 #ifdef HAVE_SECTION_QUOTES
-# define __sec_comment "\"\n\t#\""
+# define __sec_comment "\"\n#APP\n\t#\""
 #else
-# define __sec_comment "\n\t#"
+# define __sec_comment "\n#APP\n\t#"
 #endif
 #define link_warning(symbol, msg) \
   __make_section_unallocated (".gnu.warning." #symbol) \
